rifle.py

```diff
@@ -36,17 +40,49 @@ def __init__(self, universe, DRAWING_SCALE):
         self.word_length_min_ = 3
         self.word_length_range = 3
 
-
+    def getTargetID(self,input): 
+        for enemy in self.universe_.enemies(): 
+            target_tag_word = self.target_tags_[enemy.ID_]
+            if(input == target_tag_word):
```

Comment:
if statement do not need brackets in python, for good style they should not be included.